news/2018-10-12-new-featured-artist-tracks-available.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "New Featured Artist Tracks Available"
+date: 2018-10-12 15:00:00 +0000
+---
+
+We're proud to announce the latest addition to the Featured Artist catalogue — namely **three** new collections of awesome tracks, one from [\*namirin](https://osu.ppy.sh/beatmaps/artists/18), one from [Dictate](https://osu.ppy.sh/beatmaps/artists/21), and one from [LukHash](https://osu.ppy.sh/beatmaps/artists/12)!
```

Comment:
addition? additions? supplement? shipment? not sure

Reply:
maybe mention "one album from xxx" right now it sounds like "one track" from each. or just state "albums" rather than "collections"